src/utilities/theme/utils.ts

```diff
@@ -70,25 +76,25 @@ export function buildColors(theme: ThemeConfig) {
   } = UNSTABLE_colors;
   /* eslint-enable babel/camelcase */
 
-  const lightSurface = isLight(hslToRgb(colorToHsla(surface)));
+  const lightSurface = isLight(hexToRgb(surface));
 
   return {
     ...customPropertyTransformer({
-      ...surfaceColors(colorToHsla(surface), lightSurface),
-      ...onSurfaceColors(colorToHsla(onSurface), lightSurface),
-      ...interactiveColors(colorToHsla(interactive), lightSurface),
-      ...neutralColors(colorToHsla(neutral), lightSurface),
-      ...brandedColors(colorToHsla(branded), lightSurface),
-      ...criticalColors(colorToHsla(critical), lightSurface),
-      ...warningColors(colorToHsla(warning), lightSurface),
-      ...highlightColors(colorToHsla(highlight), lightSurface),
-      ...successColors(colorToHsla(success), lightSurface),
+      ...surfaceColors(hexToHsluv(surface), lightSurface),
+      ...onSurfaceColors(hexToHsluv(onSurface), lightSurface),
+      ...interactiveColors(hexToHsluv(interactive), lightSurface),
+      ...neutralColors(hexToHsluv(neutral), lightSurface),
+      ...brandedColors(hexToHsluv(branded), lightSurface),
+      ...criticalColors(hexToHsluv(critical), lightSurface),
+      ...warningColors(hexToHsluv(warning), lightSurface),
+      ...highlightColors(hexToHsluv(highlight), lightSurface),
+      ...successColors(hexToHsluv(success), lightSurface),
```

Comment:
This functionally narrows our acceptable color space API from hex, rgba, hsla, down to just hex. Is that what we want?

Reply:
We can rework this so it continues to support those 3 colors

Reply:
We can rework this so it continues to support those 3 colors